app/serializers/subject_and_class_serializer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# SubjectAndClassSerializer
+class SubjectAndClassSerializer
+  include FastJsonapi::ObjectSerializer
+
+  attributes :id, :created_at
+  attribute :subject do |object|
```

Comment:
@josephelias-rubyians  This is one way of adding the attributes. Just for learning purpose, the other way is to do as follows. This comment is just for leaning, not to make the code change.

attribute :creator do |object|
        AccountSerializer.new(object.creator)
 end

Reply:
Okay I got it.